src/main/java/org/alliancegenome/curation_api/controllers/crud/GeneExpressionAnnotationCrudController.java

```diff
@@ -16,6 +17,7 @@
 import java.util.List;
 
 @RequestScoped
+@JBossLog
```

Comment:
Remove unused @JBossLog annotation and import

Reply:
Removed